File: apzkr-pzpi-21-11-milennyi-ivan/Task3-WebClient/src/viewModels/BreedVM.ts
// Клас для відображення інформації про породу
export class BreedVM {
    id: number; // Ідентифікатор породи
    name: string; // Назва породи
    info: string; // Інформація про породу
    feedName: string; // Назва корму, який використовується для цієї породи
    sheepCount: number; // Кількість овець цієї породи

    constructor(id: number, name: string, info: string, feedName: string, sheepCount: number) {
        this.id = id;
        this.name = name;
        this.info = info;
        this.feedName = feedName;
        this.sheepCount = sheepCount;
    }
}

File: apzkr-pzpi-21-11-milennyi-ivan/Task1-Server/src/view_models/breed_vm.rs
use serde::Serialize;
use sqlx::FromRow;
use utoipa::ToSchema;

// Структура, що представляє представлення породи (ViewModel)
#[derive(Debug, Serialize, FromRow, ToSchema)]
pub(crate) struct BreedVM {
    id: u64, // Ідентифікатор породи
    name: String, // Назва породи
    info: String, // Інформація про породу
    feed_name: String, // Назва корму
    sheep_count: u64, // Кількість овець цієї породи
}
